lib/daru/core/group_by.rb

```diff
@@ -298,6 +312,31 @@ def apply_method method_type, method
         Daru::DataFrame.new(rows.transpose, index: index, order: order)
       end
 
+      # Do the `method` (`method` can be :sum, :mean, :std, :median, etc or
+      # lambda/function/Proc), on the column.
+      def apply_method_on_colmns colmns, method
```

Comment:
This method is inspired from the [apply_method ](https://github.com/SciRuby/daru/pull/340/files#diff-e2feb9bd3b453b5a4adb14512ac7e8ebR298). But this method is defined for applying `method` only on one column and return the result array.